### DISCUSSION TITLE
No scroll to error when saving within globals or collections

### DISCUSSION THREAD

**Discussion by todor0v** [payloadcms/payload#7266] [Feature Requests & Ideas]
### Describe the Bug

When saving there is no scroll to the error, making it hard to find when there are very long pages with many fields. For example if you have a required field and you cannot save the page without filling it, you do see an error message, but it would be a better UX to be scrolled to the problematic field itself.

### To Reproduce

1. Have a required field in your globals or collections and do not fill it.
2. Save the page.
3. You will see an error "Please correct invalid fields", but you have to then search where they are.

### Payload Version

2.17.0

### Adapters and Plugins

db-mongodb

**Comment by paulpopus** [comment]
Converting this to a feature suggestion, there's probably some better ways for us to display errors and make finding them easier. We can visit this for UI 3.0, copying in @tylandavis
